tests/core/views/test_api.py

```diff
@@ -367,12 +371,18 @@ async def test_pay_real_invoice(
     assert payment.payment_hash == invoice["payment_hash"]
 
     # check the payment status
-    response = await api_payment(
-        invoice["payment_hash"], inkey_headers_from["X-Api-Key"]
+    response = await client.get(
+        f'/api/v1/payments/{invoice["payment_hash"]}', headers=inkey_headers_from
     )
-    assert response["paid"]
+    assert response.status_code < 300
+    payment_status = response.json()
+    assert payment_status["paid"]
 
-    status = await WALLET.get_payment_status(invoice["payment_hash"])
+    # create a dummy payment object of which we will
+    # only use checking_id in get_payment_status
+    status = await WALLET.get_payment_status(
+        Payment.dummy(checking_id=invoice["payment_hash"])
+    )
     assert status.paid
```

Comment:
how is it possible that this test is passing on cln-rest? Get payment status requires a bolt11, but none is given here.